logicrunner/casebind.go

```diff
@@ -28,14 +28,11 @@ import (
 	"github.com/insolar/insolar/core/message"
 	"github.com/insolar/insolar/core/reply"
 	"github.com/pkg/errors"
-	"github.com/ugorji/go/codec"
 	"golang.org/x/crypto/sha3"
 )
 
 func HashInterface(in interface{}) []byte {
```

Comment:
this is a wrong function now and must be removed, please change all it's usages for core function.